src/Navigation/Search/styled.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import { ReactComponent as searchIcon } from "../../assets/icon-search.svg";
+
+export const Wrapper = styled.label`
+    margin:0 auto;
```

Comment:
I would remove that because the search bar is not centered on desktop
+ I would add `justify-content: space-between;` to Navigation/styled.js/Container to separate `<Header />` and `<Search />`

Reply:
thx for suggestion:)